object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/ObjectConstructionChecker.java

```diff
@@ -11,13 +13,19 @@
  * objects from being instantiated.
  */
 @SuppressWarningsKeys({"builder", "object.construction", "objectconstruction"})
+@SupportedOptions(ObjectConstructionChecker.USE_VALUE_CHECKER)
 public class ObjectConstructionChecker extends BaseTypeChecker {
 
+  public static final String USE_VALUE_CHECKER = "useValueChecker";
+
   @Override
   protected LinkedHashSet<Class<? extends BaseTypeChecker>> getImmediateSubcheckerClasses() {
     LinkedHashSet<Class<? extends BaseTypeChecker>> checkers =
         super.getImmediateSubcheckerClasses();
     checkers.add(ReturnsRcvrChecker.class);
+    if (this.processingEnv.getOptions().containsKey(USE_VALUE_CHECKER)) {
```

Comment:
This is nicer than the hack I was suggesting 😄

Reply:
This seems fine to me.

Reply:
Please add a brief comment about why you did it this way.